versioned_docs/version-1.1(WIP)/depAndWthd.md

```diff
@@ -0,0 +1,107 @@
+---
+id: depAndWthd
+title: Deposit and Withdrawal
+---
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+Deposit and withdrawal is a special messaging mechanism between layer 1 and layer 2 with assets transfer function. Users can deposit assets from layer1 (CKB) to layer2 (Godwoken), or withdraw assets from layer 2 to layer 1.
+
+## Deposit
+
+A deposit request is created by sending a layer1 transaction. This transaction generates a cell with a special lock - deposit lock. The block producer will collect these cells and process the deposit requests in the blocks. It will unlock the deposit cells, move the assets under the custodian lock，update layer2 state and submit layer1 transactions in the blocks. These transactions are checked by the layer1 script, so the block producer cannot take the assets away.
+
+The deposit cell:
+
+``` yaml
+lock:
+  code_hash:    (deposit lock's code hash),
+  hash_type:    Type,
+  args: (rollup_type_hash(32 bytes) | DepositLockArgs)
+capacity:   (deposit CKB),
+type_:  (none or SUDT script)
+data:   (none or SUDT amount)
+```
+
+The `lock` field of the deposit cell uses deposit lock. The first 32 bytes of `args` is a unique value associated with the rollup instance. The data structure `DepositLockArgs` denotes to which layer2 account the user deposited. The `capacity` is the total amount of CKB user deposited. The `type_` and `data` fields follow the CKB Simple UDT format. All these fields enable users to deposit Simple UDT assets to layer2.
+
+```
+table DepositLockArgs {
+    // layer1 lock hash
+    owner_lock_hash: Byte32,
+    layer2_lock: Script,
+    cancel_timeout: Uint64,
+    registry_id: Uint32,
+}
+```
+
+`DepositLockArgs` represents the script and `registry_id` of layer2 account, currently, which currently supports only the ETH registry. Users can cancel deposit after `cancel_timeout`, which can be used by block producer to reject packed deposit cell when the deposited cell contains invalid data.
```

Comment:
`DepositLockArgs` represents the script and `registry_id` of a layer 2 account, which currently supports only the ETH registry. Users can cancel a deposit after the expiration of `cancel_timeout`, which may be used by the block producer to reject packed deposit cells containing invalid data.